go.mod

```diff
@@ -25,6 +25,8 @@ require (
 	github.com/lib/pq v1.2.0
 	github.com/markphelps/flipt-grpc-go v0.0.0-20190303144529-3ebb133e62c0
 	github.com/mattn/go-sqlite3 v1.11.0
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
```

Comment:
@markphelps for whatever reason these always get inserted into the `go.mod` file when I run `make dev` or `make test`, but then go away when I run `go mod tidy`. Do you happen to know any potential causes of this? Is it due to these being used at one time and now they are lingering in the `go.sum` file?

Reply:
Hmm I'm note sure. What version of go are you running locally? When I pulled down your changes and built them it automatically pruned my `go.mod` file to remove these lines:

```
github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
github.com/modern-go/reflect2 v1.0.1 // indirect
```

Reply:
I just realized that I was on 1.12.6. Updated to 1.13.1 and these do not seem to show up anymore. Thanks!